### ISSUE TITLE
Errors on windows box

### ISSUE THREAD

**Issue by albertosantini** [FamousTools/famous-dist-generator#4]
After `npm install -g famous-dist-generator` I executed the below commands.

Any idea?

```
$ famous-dist-generator --standalone --ref 0.2.1 --minify --out ./famous-standalone-0.2.1.js                 
                                                                                                                                                          
c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\lib\index.js:170                                                                   
mo[key] = convertAMDToCommonJS(code, codeStyleOptions, resolvePath).trim('\n')                                                                            
                                                                    ^                                                                                     
TypeError: Cannot call method 'trim' of null                                                                                                              
    at c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\lib\index.js:170:77                                                         
    at Array.reduce (native)                                                                                                                              
    at c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\lib\index.js:166:40                                                         
    at c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\lib\index.js:75:5                                                           
    at c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\node_modules\async\lib\async.js:254:17                                      
    at done (c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\node_modules\async\lib\async.js:135:19)                               
    at c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\node_modules\async\lib\async.js:32:16                                       
    at c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\node_modules\async\lib\async.js:251:21                                      
    at c:\Users\icebox\AppData\Roaming\npm\node_modules\famous-dist-generator\lib\index.js:70:7                                                           
    at fs.js:271:14                                                                                                                                       
```
```
$ famous-dist-generator --standalone --ref 0.2.2 --minify --out ./famous-standalone-0.2.2.js
Failed to build standalone famous for reference 0.2.2
{ [Error: Cannot find module '....' from 'C:\Users\icebox\AppData\Local\Temp\0.2.2-1411754682433\physics\forces']
  stream:
   { _readableState:
      { highWaterMark: 16,
        buffer: [],
        length: 0,
        pipes: [Object],
        pipesCount: 1,
        flowing: true,
        ended: false,
        endEmitted: false,
        reading: true,
        sync: false,
        needReadable: true,
        emittedReadable: false,
        readableListening: false,
        objectMode: true,
        defaultEncoding: 'utf8',
        ranOut: false,
        awaitDrain: 0,
        readingMore: false,
        decoder: null,
        encoding: null,
        resumeScheduled: false },
     readable: true,
     domain: null,
     _events:
      { end: [Object],
        error: [Object],
        data: [Function: ondata],
        _mutate: [Object] },
     _maxListeners: 10,
     _writableState:
      { highWaterMark: 16,
        objectMode: true,
        needDrain: false,
        ending: true,
        ended: true,
        finished: true,
        decodeStrings: true,
        defaultEncoding: 'utf8',
        length: 0,
        writing: false,
        corked: 0,
        sync: false,
        bufferProcessing: false,
        onwrite: [Function],
        writecb: null,
        writelen: 0,
        buffer: [],
        pendingcb: 0,
        prefinished: true,
        errorEmitted: false },
     writable: true,
     allowHalfOpen: true,
     _options: { objectMode: true },
     _wrapOptions: { objectMode: true },
     _streams: [ [Object] ],
     length: 1,
     label: 'deps' } }
```

**Comment by jd-carroll** [comment]
@psyrendust @TheAlphaNerd I'm not sure we are chasing the right issues here...

First, in writeRequireJS we were still appending a 'famous' namespace even though everything has been moved to the 'src' directory. **So as of right now the famous-dist-generator is totally busted on OS's for a requirejs build.** I'll create a separate issue for this fix.

Second, writeRequireJS doesn't even use the browserify-ftw code. Further, even if the fix suggested at thlorenz/browserify-ftw#10 was picked up, we never touch the referenced code.

That said, the suggested fix for thlorenz/browserify-ftw#10 does appear to be correct. Having worked through this some, I see the modules sent to requirejs are listed as:

```
module: src\transitions\WallTransition
module: src\utilities\KeyCodes
module: src\utilities\Timer
module: src\utilities\Utility
...
```

The problem here is that requirejs optimizer doesn't know how to handle windows path separators:

```
{ [Error: SyntaxError: Unexpected token ILLEGAL
    at prim.start.then.then.eOuter.fileName (c:\Users\Joseph Carroll\AppData\Roaming\npm\node_modules\famous-dist-generator\node_modules\requirejs\bin\r.js:25458:38)
]
  originalError: 
   { [SyntaxError: Unexpected token ILLEGAL]
     fileName: 'C:/Users/JOSEPH~1/AppData/Local/Temp/git-cache/da8bff8bb308c07a06670a621b74f44c8e5a4412acbfaf71fecb27f421201b1c/raw/src\\utilities\\KeyCodes.js' } }
```

In the above exception, note the `\\` in the path. Not sure if this will completely fix the issue listed here so I will also create a separate issue for this.

@albertosantini - Can you please test once these changes are picked up. I am on Windows and everything appears to be working well for me.

**Comment by albertosantini** [comment]
It is ok. Thanks. :)

```
λ famous-dist-generator --standalone --ref 0.3.4 --minify --out ./famous.js
Successfully built standalone famous for reference 0.3.4
```
